README.md

```diff
@@ -77,6 +77,41 @@ $csv->auto('data.csv');
 print_r($csv->data);
 ```
 
+**Parse data with offset**
+* ignoring the first X (e.g. two) number of rows
```

Comment:
This is a little bit hard to read. From my perspective, removing the words "number of" fixes the sentence.